client/my-sites/upgrades/domain-management/transfer/utils.js

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
This single file with `promisy` might be a good starting point for #2607 - but it's up to you how to organize it. I just don't like generic names that don't tell what I'll find in the file. Bundling together some unrelated shared code doesn't seem like a good idea, even if just from a readability point of view. Splitting them into separate files can help with that, plus it's easier to refactor, for example, if `promisy` was to be moved somewhere else.